jobqueue/src/main/java/com/birbit/android/jobqueue/persistentQueue/sqlite/SqliteJobQueue.java

```diff
@@ -36,11 +36,13 @@
     // we keep a list of cancelled jobs in memory not to return them in subsequent find by tag
     // queries. Set is cleaned when item is removed
     Set<String> pendingCancelations = new HashSet<>();
+    FileStorage jobStorage;
     private final StringBuilder reusedStringBuilder = new StringBuilder();
     private final WhereQueryCache whereQueryCache;
 
     public SqliteJobQueue(Configuration configuration, long sessionId, JobSerializer serializer) {
         this.sessionId = sessionId;
+        jobStorage = new FileStorage(configuration.getAppContext(), "jobs_" + configuration.getId());
```

Comment:
Shouldn't the second parameter be just the id? Looking at how you use it inside `FileStorage`...